src/main/java/quickcache/model/util/SampleDataUtil.java

```diff
@@ -22,36 +23,32 @@
         Choice choice2 = new Choice("A non-compulsory module for NUS Computer Science students");
         Choice choice3 = new Choice("A module that can be S/Ued");
         Choice choice4 = new Choice("A module that doesn't involve a brown field project");
-        Choice choice5 = new Choice("Computer Organization module for NUS students");
-        Choice choice6 = new Choice("A non-compulsory module for NUS Computer Science students");
-        Choice choice7 = new Choice("A module that can be S/Ued");
-        Choice choice8 = new Choice("A module that doesn't involve assembly language");
-        Choice choice9 = new Choice("Introductory module to computer networks for NUS students");
-        Choice choice10 = new Choice("A non-compulsory module for NUS Computer Science students");
-        Choice choice11 = new Choice("A module that can be S/Ued");
-        Choice choice12 = new Choice("A module that doesn't involve TCP");
+        Choice choice5 = new Choice("bubble tea");
+        Choice choice6 = new Choice("oxygen");
+        Choice choice7 = new Choice("iphone");
+        Choice choice8 = new Choice("pepsi");
         return new Flashcard[]{
-            new Flashcard(new MultipleChoiceQuestion("What is CS2103T?",
+            new Flashcard(new OpenEndedQuestion("Sample OEQ 1 (ans: SampleAns1)",
+                    new Answer("SampleAns1")
+            ), getTagSet("OEQ", "ExampleTag")),
```

Comment:
Think we don't need this question because we have an example of an OEQ in the next question.

Reply:
Okay noted. Thanks!